client/securedrop_client/gui/main.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         # Actions
         quit = QAction(_("Quit"), self)
         quit.setIcon(QIcon.fromTheme("application-exit"))
-        quit.setShortcut(QKeySequence.Quit)
+        quit.setShortcut(Qt.CTRL + Qt.Key_Q)
```

Comment:
This means giving up [`QKeySequence`'s cross-platform defaults](https://doc.qt.io/qt-6/qt.html#ShortcutContext-enum), although `QKeySequence.Quit` seems to resolve to <kbd>Ctrl</kbd> + <kbd>Q</kbd> everywhere in practice....